src/components/common/Tree/CustomContentTree.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable react/forbid-prop-types */
```

Comment:
Remove?

Reply:
I need to use the object proptype for classes prop, and I have not seen any alternative